src/docs/product/accounts/pricing.mdx

```diff
@@ -165,7 +165,7 @@ Sentry has one free Developer plan and three paid plans: Team, Business, and Ent
 
 Plan upgrades take effect immediately. Plan downgrades and cancellations are processed as the end of the current contract cycle and cannot be refunded. If you no longer want to be on a paid plan, you'll revert to the base Developer plan at the end of your current contract.
 
-If you're currently on a paid plan, you can sign up for a 30-day trial; if you're on a free plan, you can run a trial for 14 days. Before the trial ends, Sentry will provide you options to upgrade and continue to use our latest features like Performance Monitoring. If you choose to opt in to a new plan during your trial or at the end of your trial, the new plan takes effect immediately. If you don't upgrade by the end of trial, you will be downgraded to the same plan you were on prior to starting the trial.
+You can sign up for a 14-day trial if you're on a paid plan, new Sentry accounts come with a 14-day trial by default. Before the trial ends, Sentry will provide you options to upgrade and continue to use our latest features like Performance Monitoring. If you choose to opt in to a new plan during your trial or at the end of your trial, the new plan takes effect immediately. If you don't upgrade by the end of trial, you will be downgraded to the same plan you were on prior to starting the trial.
```

Comment:
```suggestion
All new Sentry accounts come with a 14-day free trial period. You'll have the option to upgrade and continue to use Sentry's latest features like Performance Monitoring at any time before your trial ends. Upgrades will be effective immediately. Users who haven't upgraded by the end of their free trial will remain on the free Developer plan with limited features.

```

Clarifying question: Are we telling people that they get a 14-day free trial period before they have to start paying if they sign up for a paid plan? Or do they have the option to downgrade from a paid plan to a free 14-day trial period? Asking because if all accounts get a free trial period, then we don't need to make a distinction between paid plans and all other plans, and can just say "all new accounts".

Reply:
Hi @lizokm thanks for taking a look and sorry for the delay. So I only wanted to suggest this change because we don't offer a 30-day trial to anyone anymore, I just wasn't sure how much I can tweak the text. Right now, we offer a 14-day trial for everyone if they sign up, and from time to time, they can enter an additional 14-day trial, but I wouldn't really want to emphasize it - the "all new accounts" that you suggested would be perfect!

Reply:
Ok sounds great!! I'll merge this.